Disable auto-updates of the azureus platform, since we want to use debian packages
to manage updates instead.

Index: azureus-5.4.0.0/org/gudy/azureus2/core3/config/impl/ConfigurationDefaults.java
===================================================================
--- azureus-5.4.0.0.orig/org/gudy/azureus2/core3/config/impl/ConfigurationDefaults.java	2014-08-14 20:47:15.758707259 +0200
+++ azureus-5.4.0.0/org/gudy/azureus2/core3/config/impl/ConfigurationDefaults.java	2014-08-14 20:47:15.750707260 +0200
@@ -278,9 +278,9 @@
 	
 	def.put("Default save path", f.getAbsolutePath());
     
-    def.put("update.start",TRUE);
-    def.put("update.periodic",TRUE);
-    def.put("update.opendialog",TRUE);
+    def.put("update.start",FALSE);
+    def.put("update.periodic",FALSE);
+    def.put("update.opendialog",FALSE);
     def.put("update.autodownload", FALSE);
 
     def.put( "Config Verify Frequency", new Long( 30*60*1000 ));
@@ -492,7 +492,7 @@
     def.put( "locale", Locale.getDefault().toString() );
     def.put( "locale.set.complete.count", ZERO);
     def.put( "Password Confirm", null );
-    def.put( "Auto Update", TRUE );
+    def.put( "Auto Update", FALSE );
     def.put( "Alert on close", FALSE );
     def.put( "diskmanager.friendly.hashchecking", FALSE );
     def.put( "diskmanager.hashchecking.smallestfirst", TRUE );    
Index: azureus-5.4.0.0/org/gudy/azureus2/pluginsimpl/update/PluginUpdatePlugin.java
===================================================================
--- azureus-5.4.0.0.orig/org/gudy/azureus2/pluginsimpl/update/PluginUpdatePlugin.java	2014-08-14 20:47:15.758707259 +0200
+++ azureus-5.4.0.0/org/gudy/azureus2/pluginsimpl/update/PluginUpdatePlugin.java	2014-08-14 20:47:15.750707260 +0200
@@ -93,7 +93,7 @@
 		
 		final PluginConfig	plugin_config = plugin_interface.getPluginconfig();
 		
-		boolean enabled = plugin_config.getPluginBooleanParameter( "enable.update", true );
+		boolean enabled = plugin_config.getPluginBooleanParameter( "enable.update", false );
 
 		model.setConfigSectionID(PLUGIN_CONFIGSECTION_ID);
 		model.getStatus().setText( enabled?"Running":"Optional checks disabled" );
@@ -122,7 +122,7 @@
 		
 		BasicPluginConfigModel config = ui_manager.createBasicPluginConfigModel(ConfigSection.SECTION_PLUGINS, PLUGIN_CONFIGSECTION_ID);
 		
-		config.addBooleanParameter2( "enable.update", "Plugin.pluginupdate.enablecheck", true );
+		config.addBooleanParameter2( "enable.update", "Plugin.pluginupdate.enablecheck", false );
 				
 		plugin_interface.addEventListener(
 			new PluginEventListener()
Index: azureus-5.4.0.0/com/aelitis/azureus/ui/swt/UIConfigDefaultsSWTv3.java
===================================================================
--- azureus-5.4.0.0.orig/com/aelitis/azureus/ui/swt/UIConfigDefaultsSWTv3.java	2014-08-14 20:47:15.758707259 +0200
+++ azureus-5.4.0.0/com/aelitis/azureus/ui/swt/UIConfigDefaultsSWTv3.java	2014-08-14 20:47:15.754707259 +0200
@@ -142,7 +142,7 @@
 
 			defaults.addParameter("window.maximized", true);
 
-			defaults.addParameter("update.autodownload", true);
+			defaults.addParameter("update.autodownload", false);
 			
 			//defaults.addParameter("suppress_file_download_dialog", true);
 			
